### DISCUSSION TITLE
$nuxt Type Error

### DISCUSSION THREAD

**Discussion by stephenjason89** [vuejs/pinia#1526] [Help and Questions] [answered]
### Reproduction

Install @pinia/nuxt 0.3.1 together with pinia 2.0.17

### Steps to reproduce the bug

use this.$nuxt inside a pinia store
<img width="1118" alt="image" src="https://user-images.githubusercontent.com/3810177/182923746-f6f0a0f5-1b34-475e-aeef-e55bf8740c1e.png">

tsconfig already has the types but still throws error
<img width="281" alt="image" src="https://user-images.githubusercontent.com/3810177/182923917-aa3fc361-aae4-4b30-a811-5f0b6bc0c5fc.png">

### Expected behavior

no errors

### Actual behavior

IDE errors

**Comment by antlionguard** [comment, marked as answer]
This was documented at README.md for nuxt version.

Check the 0.3.0 breaking changes section.

https://github.com/vuejs/pinia/blob/v2/packages/nuxt/CHANGELOG.md

**Comment by stephenjason89** [comment]
@antlionguard Thank you for the link. It was suggested to replace it with useNuxtApp and avoid using the old $nuxt. But in my case, I rolled back to the old $nuxt because I was getting 
<img width="620" alt="image" src="https://user-images.githubusercontent.com/3810177/183125857-c22b8356-980e-4bd1-8035-df9eb7397608.png">

I changed this
<img width="857" alt="image" src="https://user-images.githubusercontent.com/3810177/183125974-b5f3a624-4874-430e-8a9e-2639497aefc0.png">

to this
<img width="861" alt="image" src="https://user-images.githubusercontent.com/3810177/183126044-346e19ac-1422-4c9a-9968-a194b90419c9.png">

Until it gets fixed.

I will be using ts-ignore for now instead of installing @nuxt/types 


This part of the code upon page load is being executed in the server side. Which I believe causes the error.

useNuxtApp() & useRuntimeConfig not available inside the store during SSR?

Thank you

**Reply by antlionguard** [reply]
It should be. Maybe you can create new issue for that.

**Reply by stephenjason89** [reply]
Will do @antlionguard 
Thank you 😊

**Comment by stephenjason89** [comment]
Here's a link to the issue @antlionguard 
https://github.com/nuxt/bridge/issues/467
